cmd/queue/main.go

```diff
@@ -425,7 +429,7 @@ func pushRequestLogHandler(currentHandler http.Handler) http.Handler {
 	return handler
 }
 
-func pushRequestMetricHandler(currentHandler http.Handler) http.Handler {
+func pushRequestMetricHandler(currentHandler http.Handler, component string) http.Handler {
```

Comment:
Currently `pkg/metrics` lib only allows one exporter for a binary. New call of this func will override existing exporter. It won't produce both `app_request_count` and `revision_request_count`.

Reply:
Fixed as we discussed offline--use the same exporter and different metrics.  In particular, we will have metrics: revision_request_count and revision_app_request_count (and similarly for latencies).